site/zenodo_rdm/moderation/handlers.py

```diff
@@ -77,17 +81,20 @@ def run(self, identity, draft=None, record=None, user=None, uow=None):
                 )
                 return
 
-            score = 0
-            for rule in self.rules:
-                score += rule(identity, draft=draft, record=record)
+            results = {}
+            for name, rule in self.rules.items():
+                results[name] = rule(identity, draft=draft, record=record)
 
+            evaluation = self.evaluate_result(results)
             action_ctx = {
                 "user_id": user.id,
                 "record_pid": record.pid.pid_value,
-                "score": score,
+                "results": results,
+                "final_evaluation": evaluation,
```

Comment:
```suggestion
                "evaluation": evaluation,
```

nit: you never know when the "final" evaluation comes :)